packages/core/README.md

```diff
@@ -4,45 +4,51 @@
 
 ## Usage
 
-#### Inputs/Outputs
-
-You can use this library to get inputs or set outputs:
+### Import the package
 
 ```js
+// javascript
 const core = require('@actions/core');
 
-const myInput = core.getInput('inputName', { required: true });
+// typescript
+import * as core from '@actions/core';
+```
 
-// Do stuff
+#### Inputs/Outputs
+
+Action inputs can be read with `getInput`.  Outputs be set with `setOutput` which makes them available to be mapped into inputs of other actions to ensure they are decoupled.
```

Comment:
"Outputs **can** be set" would be preferred
Spacing between `GetInput`. and Outputs has some extra spaces.